lib/release_notes_bot/persists.ex

```diff
@@ -19,6 +19,10 @@ defmodule ReleaseNotesBot.Persists do
     parent_id = space_parent_id
     user = Application.get_env(:release_notes_bot, :confluence_email)
     apikey = Application.get_env(:release_notes_bot, :confluence_api_key)
+    organization = "mojotech"
```

Comment:
just a thought, is this something that we can set in an Env var? or is something like that coming later?

Reply:
Yes, this has come in [PR32](https://github.com/mojotech/captains-log-release-agent/pull/32/files#diff-9875aef6f6713ad6e0a9a7ce3e1ada4b084284e5415101b3a202472b9562ad84)